com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -1045,7 +1049,7 @@ private static void logErrorAndExit(Exception e) {
 
     public static void main(String[] args) throws IOException, CompileExceptionError, URISyntaxException, LibraryException {
         try {
-            boolean success = invoke(null, new ConsoleProgress(), false, null, args).success;
+            boolean success = invoke(null, new ConsoleProgress(), null, args).success;
```

Comment:
I removed `fromEditor` TimeProfiler initializer arg. Now time profiling only measures the bob invocation time.